Documentation/About/Changelog/4-1-0.rst

```diff
@@ -0,0 +1,26 @@
+.. include:: ../../Includes.txt
+
+==========================
+Version 4.1.0 - 2020/06/03
+==========================
+
+This release is a feature release. It contains the implementation of OAuth2 authorization protocol.
+
+Download
+========
+
+Download this version from the `TYPO3 extension repository <https://extensions.typo3.org/extension/mautic/>`__ or from
+`GitHub <https://github.com/mautic/mautic-typo3/releases/tag/v4.1.0>`__.
+
+Added
+=====
+* Added support for OAuth2, thus you can now choose between OAuth1 and OAuth2. Next major version will drop OAuth1 support.
```

Comment:
When you announce to drop OAuth1 support:

- Mark all code used for OAuth1 handling as deprecated and trigger deprecation warnings
- Add this information to the "Deprecated" section of this change log entry
- Provide a guide on how to migrate extension configuration when switching from OAuth1 to OAuth2